assets/css/layouts/_authentication.scss

```diff
@@ -0,0 +1,17 @@
+.layout-authentication {
```

Comment:
For layout's file name, we should remove the `_` prefix as in [compass](https://nimblehq.co/compass/development/code-conventions/css/#stylesheets-structure) 👍